modules/electrophysiology_browser/jsx/components/electrophysiology_session_panels.js

```diff
@@ -555,16 +560,85 @@ class FilePanel extends Component {
       </Panel>
     );
   }
+
+    /**
+     * Called by download all or download annotations button to
+     * update the annotation files before downloading
+     */
+    handleClick() {
+        console.log('The link was clicked.');
+        console.log(this.state.id);
+
+        fetch(this.props.url, {
+            method: 'PUT',
```

Comment:
I believe PUT is semantically incorrect: https://www.guru99.com/put-vs-post.html#5
This operation is not idempotent.